hpx/runtime/actions/basic_action.hpp

```diff
@@ -715,8 +715,7 @@ namespace hpx { namespace actions
 #define HPX_ACTION_USES_HUGE_STACK(action)                                    \
     HPX_ACTION_USES_STACK(action, threads::thread_stacksize_huge)             \
 /**/
-#define HPX_ACTION_DOES_NOT_SUSPEND(action)                                   \
-    HPX_ACTION_USES_STACK(action, threads::thread_stacksize_nostack)          \
+#define HPX_ACTION_DOES_NOT_SUSPEND(action)
```

Comment:
Should we turn this into a warning saying that the feature has been removed and the macro will be removed eventually as well?